### DISCUSSION TITLE
[Feature] Make distance between years in timeline evenly spaced, regardless of included assets

### DISCUSSION THREAD

**Discussion by Sums123** [immich-app/immich#7988] [Feature Request]
### Timeline spacing settings

I couldn't find an open discussion regarding this.

My preference for the timeline would be that the space between each year is evenly spaced. It would be great to have an option for both variants

![image](https://github.com/immich-app/immich/assets/29928499/4d42dd70-05c0-4a63-96d8-3ab47a0d17da)

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by mertalev** [comment]
The logic for this timeline is complicated. I don't think it's worth maintaining two separate implementations for it.
